table.cc

```diff
@@ -2446,6 +2449,30 @@ table::run_with_compaction_disabled(std::function<future<> ()> func) {
     });
 }
 
+bool
+table::set_auto_compaction(bool enabled) {
+    if (enabled) {
```

Comment:
This is a good sign that this should be two functions instead:
`enable_auto_compaction()`
`disable_auto_compaction()`

Reply:
ok

Reply:
done